### PR TITLE
und-801

### PR DESCRIPTION
**Pull request by Ashish-srij**
header issue fixed

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const CountrySiteHeader = ({
               data={languageswitcherData.languagedata}
               headerText={languageswitcherData.headerText}
             />
-            <button className='menu-icon'></button>
+            <button className='menu-icon'>mobile bar</button>
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx:42]: @Ashish-srij Kindly add animation for Hamburger Menu. I do not see it implemented. Kindly check the following link for the example:
https://undp-design-system.azurewebsites.net/showcase/buttons
image

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx [hunk 2]
```diff
@@ -30,11 +30,11 @@ const GlobalHeader = ({
               data={languageswitcherData.languagedata}
               headerText={languageswitcherData.headerText}
             />
-            <button className='menu-icon'></button>
+            <button className='menu-icon'>mobile bar</button>
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx:33]: @Ashish-srij Any specific reason this text is required? If not can you remove it?

Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx:33]: @shishirphadke blank space is creating a violation issue.

Comment (shishirphadke) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx:33]: Ok

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx:42]: @shishirphadke animation is not mentioned on requirement. Do we need to add it on lot 1.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx (hunk 2 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx]: @shishirphadke Done.

**Pull request merged**